src/analysis/souffle/tests/arcs_fact_tests/is_member_of_unit_test.dl

```diff
@@ -0,0 +1,55 @@
+#include "taint.dl"
+#include "fact_test_helper.dl"
+
+claimHasTag("R.foo.a", "userSelection").
+edge("R.foo.a", "R.bar.a").
+edge("R.foo.b", "R.bar.b").
+edge("R.foobaz.a", "R.bar.c").
```

Comment:
Are these needed for this test?

Reply:
They basically are just there to introduce access paths in a fashion that is more like what we might see in the wild. I can turn them into `isAccessPath` if that would make things more clear.

Reply:
Added a comment explaining the purpose of the edges and claims.

Reply:
Converting them into `isAccessPath` + adding a  comment is cleaner? Don't feel strongly about this though.

Reply:
Acknowledged, switched the test to do that instead.